### PR TITLE
Package repository tweaks

### PR DESCRIPTION
**Pull request by imiric**
These are a few changes I did while copying packages from Bintray to https://dl.k6.io/. The biggest one is the switch back to `s3cmd` for S3 operations since [aws-cli doesn't preserve timestamps when syncing with S3](https://github.com/aws/aws-cli/issues/3069#issuecomment-818732309), and s3cmd does this nicely with object metadata. It also has better mimetype detection and checks file equality with MD5 hashes instead of relying on file size or `mtime`. So now we use aws-cli only for CloudFront invalidation and s3cmd for everything else.

I'll leave the original WIP branch (`ci/populate-dl-k6-io`) unmerged, since it has the script to download from Bintray and some additional tweaks, in case we need it again soon, and we can delete it in a few weeks.

### CODE REVIEW

**Review by MStoykov — DISMISSED** [review]

File: packaging/Dockerfile
Comment (imiric) [inline packaging/Dockerfile:10]: I'll change this to pin and expose `S3CMD_VERSION` just like `AWSCLI_VERSION` is exposed now.

Comment (imiric) [reply packaging/Dockerfile]: Done in a35bbfab.

**Review by MStoykov — DISMISSED** [review]

**Review by MStoykov — DISMISSED** [review]

**Comment by imiric** [comment]
@MStoykov @na-- See d39bb943 with the RPM repo package changes.

I did this in a hurry, cobbled together from several sources ([this](https://wiki.eprints.org/w/Creating_an_Eprints_RPM) was most helpful), but this works now:

```
yum install -y https://dl.k6.io/rpm/repo.rpm
yum install -y k6
```

Let's not merge this yet, I want to test it better on Monday.

**Review by MStoykov — APPROVED** [review]

**Review by na-- — APPROVED** [review]
LGTM, though I still haven't actually tested dl.k6.io packages all that much, I think we can merge this and make another PR if there are any other fixes that need to be done

**Pull request merged**
